tensorflow/contrib/go/lib.go

```diff
@@ -0,0 +1,6 @@
+package tensorflow
+
+// #cgo LDFLAGS: -L${SRCDIR}/../../../bazel-bin/tensorflow -ltensorflow
```

Comment:
I didn't try to do a go get, I'll try. I supposed that since go get clones the repo it must be, but not that I think, perhaps it doesn't clones using --recursive . In that case I think that is better to add a Install section on the README.md